aviary/variable_info/variables.py

```diff
@@ -604,61 +601,72 @@ class Wing:
 
 
 class Dynamic:
-    """Dynamic mission data hierarchy"""
+    """All time-dependent variables used during mission analysis"""
+
+    class Atmosphere:
+        """Variables related to atmospheric/freestream conditions"""
 
-    class Mission:
-        # all time-dependent variables used during mission analysis
         ALTITUDE = 'altitude'
         ALTITUDE_RATE = 'altitude_rate'
-        ALTITUDE_RATE_MAX = 'altitude_rate_max'
-        BATTERY_STATE_OF_CHARGE = 'battery_state_of_charge'
-        CUMULATIVE_ELECTRIC_ENERGY_USED = 'cumulative_electric_energy_used'
         DENSITY = 'density'
+        DYNAMIC_PRESSURE = 'dynamic_pressure'
+        MACH = 'mach'
+        MACH_RATE = 'mach_rate'
+        SPEED_OF_SOUND = 'speed_of_sound'
+        STATIC_PRESSURE = 'static_pressure'
+        TEMPERATURE = 'temperature'
+        VELOCITY = 'velocity'
+        VELOCITY_RATE = 'velocity_rate'
+
+    class Mission:
```

Comment:
I don't see a strong distinction between Dynamic.Mission and Dynamic.Vehicle. I'd be fine with consolidating the items here with Dynamic.Vehicle. 

Something like ` CUMULATIVE_ELECTRIC_ENERGY_USED` is probably just the integral of 'ELECTRIC_POWER_IN_TOTAL' which is being kept under Dynamic.Vehicle.Propulsion. Would we ever use battery power not to propel the vehicle? if so maybe Dynamic.Vehicle could work.

Isn't Dynamic.Mission.Distance just a description of how far downrange the vehicle is? sounds like a "vehicle state" to me.

Reply:
New Plan:
Mission becomes kinematic description of vehicle states in a ground-fixed axis i.e. 6DOF motions and constraints.
These values are typically ingested by the Equations of Motion to determine vehicle state at a later time.
# TODO: Vehicle summary forces in X,Y,Z and summary torques in X,Y,Z and should also go here
Things that go here:
Altitudes
Velocities
Angle of Attack
Distances
Flight Path Angle